### DISCUSSION TITLE
How to write and generate a webhook？

### DISCUSSION THREAD

**Discussion by AllenZMC** [clusterpedia-io/clusterpedia#535] [General]
How to write and generate a webhook for the AA？

**Comment by Iceber** [comment]
The logic for implementing a Webhook for AA resources is the same as for other resources, but you need to enable Admission in the AA server. 
eg.
https://github.com/clusterpedia-io/clusterpedia/blob/b88e2675ecacb47a24465efe3f1774ffd36f8d30/cmd/apiserver/app/options/options.go#L140
